Admin/Model/UserAdmin.php

```diff
@@ -209,8 +209,8 @@ protected function configureFormFields(FormMapper $formMapper)
                         'dp_max_date' => $now->format('c'),
                         'required' => false,
                     ])
-                    ->add('firstname', null, ['required' => false])
```

Comment:
Please remove the pre-sf2.8 hacks too:
https://github.com/ebuildy/SonataUserBundle/blob/195c5848cad92f0b67a73922b4cdb0fadb775bd7/Admin/Model/UserAdmin.php#L176-L194